source/tutorials/tls.rst

```diff
@@ -145,13 +145,14 @@ Client Setup
 ~~~~~~~~~~~~
 
 The client setup is equally simple. You need less certificates, just the
-CA cert. 
+CA & client cert. 
```

Comment:
We need to change this so that folks know they need the client cert only if the actually want authentication (and not anon mode).